data/Britton_1994/metadata.yml

```diff
@@ -0,0 +1,229 @@
+    pieces, mixed well, and stored in calico-lined aluminium bins in the dark. Storage
+    temperatures over the 12-month experimental period ranged from a minimum of lS°C
+    in winte, to a maximum of 2S°C in summer. Because seeds are not homogeneously
+    distributed in space, pooling and mixing of the original samples was carried out
+    in order to reduce variability associated with collecting from different areas
+    of each wetland.
+  original_file: none, transcribed from manuscript
+  notes: none
+locations:
+  northern New South Wales:
+    latitude (deg): -30.65
+    longitude (deg): 151.5
+    description: five wetlands (locally named 'lagoons') on which this study is based
+      are located at altitudes between 1000 and 1325 m on the New England plateau
+      in northern New South Wales
+contexts: .na
+traits:
+- var_in: germination_glasshouse
+  unit_in: .na
+  trait_name: recruitment_time
+  entity_type: population
+  value_type: mode
+  basis_of_value: measurement
+  replicates: 30
+  basis_of_record: lab
+  methods: Four germination trials were run in seasonal succession, commencing in
+    September 1992 (spring 1) and ending in August 1993 (winter). A fifth trial (spring
+    2) was conducted in September 1993 in order to assess whether dry storage of the
+    sediments over a 12-month period had any influence on germination response. Dry
+    storage may increase the sensitivity of seeds to  emperature and hence increase
+    the likelihood of germination (Thompson and Grime 1983; Probert et al. 1987).
+    In each trial, 125-mL subsamples of dried sediment were placed in clear plastic
+    trays (16 X 10 X 5 cm), watered with tap water, and maintained in a waterlogged
+    condition for a period of eight weeks. Thirty trays (six sites x five replicates)
+    were placed at random within a shallow, Polythene-lined tank situated outdoors
+    in the grounds of the University, and thirty (six sites X five replicates) were
+    placed within a tank inside a glasshouse. The trays were perforated at their bases
+    to allow movement of water from the tanks into the trays. Artificial lighting
+    or heating was not used in the glasshouse, although a cooling system, which came
+    into operation at around 27OC, maintained temperatures in the glasshouse below
+    30°C for most of the year. At the end of the eight-week wetting period, individuals
+    that had germinated were identified and counted. The sediments were then discarded.
+    Maximum-minimum thermometers were used to measure air temperatures and water temperatures
+    within the tanks, both in the glasshouse and outside. Readings were taken at approximately
+    three-day intervals throughout each germination trial. Light measurements were
+    recorded at midday at the same intervals with the aid of a Li-Cor (LI-185B) photometer,
+    which measures radiation in the 400 to 700 nm waveband. Data obtained in the first
```

Comment:
Maybe remove the rest of the methods from this sentence (and for the trait below)?

Reply:
Done